_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -419,6 +419,18 @@ factors. Refer to
 link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_physical.html#enable-hw-mfa-for-root[
 the documentation for more information on using a hardware device with the root user].
 
+[[subscribe_sns]]
+===== Subscribe to SNS topic
+The Config alerts and CloudWatch Metric Alarms all go to an SNS topic. Unfortunately, there is no way to automate
+subscribing to the SNS topic as each of the steps require validating the delivery target. Follow the steps outlined in
+the link:https://docs.aws.amazon.com/sns/latest/dg/sns-user-notifications.html[AWS docs] for configuring notifications
+to you for each of the alerts.
```

Comment:
NIT: Should this be "configuring notifications *for* you"?

Reply:
I decided to skirt the issue by being more explicit: "Follow the steps ... to be notified by Email, Phone, or SMS for each of the alerts"